### PR TITLE
add more students

### PR DESCRIPTION
**Pull request by tarasglek**
+ doc

### CODE REVIEW

**Review by humphd — APPROVED** [review]
@Rachit1313 do you want to pull this branch locally and try decrypting this?  It should work now that your key is included.

**Comment by Amnish04** [comment]
@tarasglek I just added an ssh key to my github account. Could you please include it as well.

**Comment by rjwignar** [comment]
@tarasglek My apologies. I also added an SSH key to my GitHub account earlier but forgot to mention it. Could you also include my key?

**Comment by tarasglek** [comment]
added more keys, landing.

File: sops/.sops.yaml [hunk 1]
```diff
@@ -1,7 +1,12 @@
 creation_rules:
   - key_groups:
       - age:
-          # humphd
-          - age19j4d6v9j7rx5fs629fu387qz4zmlpsqjexa4s08tkfrrmfdl5cwqjlaupd
-          # tarasglek
-          - age13runq29jhy9kfpaegczrzttykerswh0qprq59msgd754yermtfmsa3hwg2
+          # Generated by `/usr/local/bin/github-to-sops --github-users tarasglek,humphd,Amnish04,Rachit1313,kliu57,mingming-ma,WangGithub0,rjwignar --inplace-edit .sops.yaml` https://github.com/tarasglek/github-to-sops
```
Comment (humphd) [inline sops/.sops.yaml:4]: @rjwignar have you got a proper SSH key added on GitHub?  Looks like it didn't pull yours down for some reason

Comment (rjwignar) [reply sops/.sops.yaml:4]: Uh oh. I think I generated a SHA256 key by mistake. I'm going to upload a new key momentarily

Comment (humphd) [reply sops/.sops.yaml:4]: I made the same mistake, you're in good company!

Comment (rjwignar) [reply sops/.sops.yaml:4]: I added another SSH key just now.
I made sure to use `ssh-keygen -t ed25519 -C "[my primary GitHub email]"` this time, assuming that's the issue.

Comment (humphd) [reply sops/.sops.yaml:4]: Fix in https://github.com/tarasglek/chatcraft.org/pull/329

**Pull request merged**